inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/page/MonitoringPage.java

```diff
@@ -0,0 +1,524 @@
+import org.apache.wicket.spring.injection.annot.SpringBean;
+import org.apache.wicket.util.string.StringValue;
+import org.wicketstuff.annotation.mount.MountPath;
+
+import com.googlecode.wicket.kendo.ui.form.datetime.AjaxDatePicker;
+
+import de.tudarmstadt.ukp.clarin.webanno.api.DocumentService;
+import de.tudarmstadt.ukp.clarin.webanno.api.ProjectService;
+import de.tudarmstadt.ukp.clarin.webanno.model.AnnotationDocument;
+import de.tudarmstadt.ukp.clarin.webanno.model.AnnotationDocumentState;
+import de.tudarmstadt.ukp.clarin.webanno.model.Project;
+import de.tudarmstadt.ukp.clarin.webanno.model.SourceDocument;
+import de.tudarmstadt.ukp.clarin.webanno.security.UserDao;
+import de.tudarmstadt.ukp.clarin.webanno.security.model.User;
+import de.tudarmstadt.ukp.clarin.webanno.ui.core.menu.MenuItemRegistry;
+import de.tudarmstadt.ukp.clarin.webanno.ui.core.page.ApplicationPageBase;
+import de.tudarmstadt.ukp.inception.workload.dynamic.support.DataProvider;
+import de.tudarmstadt.ukp.inception.workload.dynamic.support.Filter;
+import de.tudarmstadt.ukp.inception.workload.dynamic.support.ImagePanel;
+import de.tudarmstadt.ukp.inception.workload.dynamic.support.ModalPanel;
+import de.tudarmstadt.ukp.inception.workload.dynamic.support.WorkloadProperties;
+
+
+@MountPath("/workload.html")
+public class MonitoringPage extends ApplicationPageBase
+{
+
+    private static final long serialVersionUID = 1180618893870240262L;
+
+    //The Icons
+    private static final ResourceReference META =
+        new PackageResourceReference(MonitoringPage.class, "book_open.png");
+
+    //Top Form
+    Form workload;
+
+    //Default Annotations value
+    private IModel<Integer> defaultAnnotations = new Model<>();
+
+    //Current Project
+    private IModel<Project> currentProject = new Model<>();
+
+    //Datatable
+    private DataTable table;
+
+    //SpringBeans
+    private @SpringBean UserDao userRepository;
+    private @SpringBean ProjectService projectService;
+    private @SpringBean MenuItemRegistry menuItemService;
+    private @SpringBean DocumentService documentService;
+    private @SpringBean WorkloadProperties workloadProperties;
+
+
+
+    //Default constructor, no project selected (only when workload.html
+    // put directly in the browser without any parameters)
+    public MonitoringPage() {
+        super();
+        //Error, user is returned to home page, nothing else to do
+        error("No Project selected, please enter the monitoring page only with a valid project reference");
+        setResponsePage(getApplication().getHomePage());
+
+    }
+
+
+
+    //Constructor with a project
+    public MonitoringPage(final PageParameters aPageParameters)
+    {
+        super(aPageParameters);
+
+        //top form initialize
+        workload = new Form("workload");
+        workload.setOutputMarkupId(true);
+
+        //Get current Project
+        StringValue projectParameter = aPageParameters.get(PAGE_PARAM_PROJECT_ID);
```

Comment:
We do not need to support the case where the project is not yet on this new page. 
If no project is set or the user has no permission to access it, the user should be returned to the default page.
Otherwise, the page can proceed with setting itself up normally.